sinfonier-web/app/views/Profile/Template/index.html

```diff
@@ -58,8 +58,10 @@ <h1 class="page-title">&{'Profile'}</h1>
           <div class="col-xs-6">
             <p>
               <span class="bold">&{'Profile.web'}:</span>
-              <a id="profile-a-web" href="${user.getImplementation().getWeb().slugify().raw()}" target="_blank">
```

Comment:
I think that with this ${user.getImplementation().getWeb()?.slugify().raw()} must be enough.